asapdiscovery-ml/asapdiscovery/ml/models.yaml

```diff
@@ -0,0 +1,7 @@
+# This file is used to specify the path to the weights file for each model.
+
+model1:
+  # type of the model, must be one of the following: GAT, ....
+  type: GAT
+  # Path to the weights file for the model1.
+  weights: https://asap-discovery-ml-weights.s3.amazonaws.com/test/model.pth
```

Comment:
would a `last_updated` tag be helpful here, or is it enough to just grab that from the S3 dashboard when we need it?

Reply:
it would be like a `datetime`? Happy to add that in.